server/utils/loggers.ts

```diff
@@ -0,0 +1,81 @@
+import winston, { format } from "winston"
+import DailyRotateFile from "winston-daily-rotate-file"
+
+const { printf, combine, timestamp, colorize, json, prettyPrint } = format
+const config = useRuntimeConfig()
+
+const path = "logs/"
+
+const rootLogger = winston.createLogger({
+  // emerg: 0,
+  // alert: 1,
+  // crit: 2,
+  // error: 3,
+  // warning: 4,
+  // notice: 5,
+  // info: 6,
+  // debug: 7
+  levels: winston.config.syslog.levels,
+  level: config.logLevel,
+  format: combine(
+    timestamp(),
+    json(),
+  ),
+  transports: [
+
+    // new winston.transports.File({
+    //   filename: path + "logfile.log",
+    // }),
+
+    new DailyRotateFile({
+      filename: path + "logfile-%DATE%.log",
+      datePattern: "YYYY-MM-DD",
+      // zippedArchive: true,
+      maxSize: "20m",
+      maxFiles: "14d",
+    }),
+
+  ],
+})
+
+// only log to terminal in development
```

Comment:
I think we should also log to the console in production since it is helpful to see the live output in docker. So we should discuss this before merging, I am just blocking this for now.